### ISSUE TITLE
Замечание

### ISSUE THREAD

**Issue by VladimirBogomolov** [Fokines/test-task-cassandra#1]
Можете обеспечить доступ к контейнеру извне не с помощью создания вилана, а с помощью средств самого докера ?

**Comment by Fokines** [comment]
Разве создание сети в докере это не средства докера?

**Comment by VladimirBogomolov** [comment]
Да, все верно, но в таком режиме другим сетевым устройствам контейнеры не видны как отдельные хосты.

**Comment by Fokines** [comment]
Понял, но как это сделать я, увы, не знаю
